src/splitInteger.test.js

```diff
@@ -4,18 +4,26 @@ const splitInteger = require('./splitInteger');
 
 test(`should split a number into equal parts
   if a value is divisible by a numberOfParts`, () => {
+  const result = splitInteger(8, 4);
 
+  expect(result).toEqual([2, 2, 2, 2]);
```

Comment:
could you please check it one more time?

Reply:
Yes, I checked - everything works properly

Reply:
Yes, I checked - everything works properly